ambry-messageformat/src/main/java/com.github.ambry.messageformat/BlobStoreHardDelete.java

```diff
@@ -93,6 +93,7 @@ public MessageInfo getMessageInfo(Read read, long offset, StoreKeyFactory storeK
             return new MessageInfo(key, header.capacity() + key.sizeInBytes() + headerFormat.getMessageSize(), true,
                 updateRecord.getAccountId(), updateRecord.getContainerId(), updateRecord.getUpdateTimeInMs());
           default:
+            // TODO (TTL update): handle TTL update
```

Comment:
why not handle this right away? Are there any unknowns?

Reply:
Need to figure out if `MessageInfo` needs to change for this. First making index changes and then figuring out what needs to be done here (since the index/hard delete is the primary consumer here)